test/midi-util-test.js

```diff
@@ -0,0 +1,320 @@
+	t.equal(pitchFromNote('g6'), 91, 'Pitch of g6 is 91');
+	t.equal(pitchFromNote('g#6'), 92, 'Pitch of g#6 is 92');
+	t.equal(pitchFromNote('a6'), 93, 'Pitch of a6 is 93');
+	t.equal(pitchFromNote('a#6'), 94, 'Pitch of a#6 is 94');
+	t.equal(pitchFromNote('b6'), 95, 'Pitch of b6 is 95');
+	t.equal(pitchFromNote('c7'), 96, 'Pitch of c7 is 96');
+	t.equal(pitchFromNote('c#7'), 97, 'Pitch of c#7 is 97');
+	t.equal(pitchFromNote('d7'), 98, 'Pitch of d7 is 98');
+	t.equal(pitchFromNote('d#7'), 99, 'Pitch of d#7 is 99');
+	t.equal(pitchFromNote('e7'), 100, 'Pitch of e7 is 100');
+	t.equal(pitchFromNote('f7'), 101, 'Pitch of f7 is 101');
+	t.equal(pitchFromNote('f#7'), 102, 'Pitch of f#7 is 102');
+	t.equal(pitchFromNote('g7'), 103, 'Pitch of g7 is 103');
+	t.equal(pitchFromNote('g#7'), 104, 'Pitch of g#7 is 104');
+	t.equal(pitchFromNote('a7'), 105, 'Pitch of a7 is 105');
+	t.equal(pitchFromNote('a#7'), 106, 'Pitch of a#7 is 106');
+	t.equal(pitchFromNote('b7'), 107, 'Pitch of b7 is 107');
+	t.equal(pitchFromNote('c8'), 108, 'Pitch of c8 is 108');
+	t.equal(pitchFromNote('c#8'), 109, 'Pitch of c#8 is 109');
+	t.equal(pitchFromNote('d8'), 110, 'Pitch of d8 is 110');
+	t.equal(pitchFromNote('d#8'), 111, 'Pitch of d#8 is 111');
+	t.equal(pitchFromNote('e8'), 112, 'Pitch of e8 is 112');
+	t.equal(pitchFromNote('f8'), 113, 'Pitch of f8 is 113');
+	t.equal(pitchFromNote('f#8'), 114, 'Pitch of f#8 is 114');
+	t.equal(pitchFromNote('g8'), 115, 'Pitch of g8 is 115');
+	t.equal(pitchFromNote('g#8'), 116, 'Pitch of g#8 is 116');
+	t.equal(pitchFromNote('a8'), 117, 'Pitch of a8 is 117');
+	t.equal(pitchFromNote('a#8'), 118, 'Pitch of a#8 is 118');
+	t.equal(pitchFromNote('b8'), 119, 'Pitch of b8 is 119');
+	t.equal(pitchFromNote('c9'), 120, 'Pitch of c9 is 120');
+	t.equal(pitchFromNote('c#9'), 121, 'Pitch of c#9 is 121');
+	t.equal(pitchFromNote('d9'), 122, 'Pitch of d9 is 122');
+	t.equal(pitchFromNote('d#9'), 123, 'Pitch of d#9 is 123');
+	t.equal(pitchFromNote('e9'), 124, 'Pitch of e9 is 124');
+	t.equal(pitchFromNote('f9'), 125, 'Pitch of f9 is 125');
+	t.equal(pitchFromNote('f#9'), 126, 'Pitch of f#9 is 126');
+	t.equal(pitchFromNote('g9'), 127, 'Pitch of g9 is 127');
+	t.equal(pitchFromNote('g#9'), 128, 'Pitch of g#9 is 128');
+	t.equal(pitchFromNote('a9'), 129, 'Pitch of a9 is 129');
+	t.equal(pitchFromNote('a#9'), 130, 'Pitch of a#9 is 130');
+	t.equal(pitchFromNote('b9'), 131, 'Pitch of b9 is 131');
+	t.equal(pitchFromNote('c10'), 132, 'Pitch of c10 is 132');
+
+	t.end();
+});
+
+test('Midi -> Util -> ensureMidiPitch', function(t) {
+	var ensureMidiPitch = Util.ensureMidiPitch;
+
+	t.equal(ensureMidiPitch(2), 2, 'Number input is accepted');
+	t.equal(ensureMidiPitch('c3'), 48, 'A string of note name and octave is accepted');
+
+	t.end();
+});
+
+test('Midi -> Util -> noteFromMidiPitch', function(t) {
+	var noteFromMidiPitch = Util.noteFromMidiPitch;
+
+	t.equal(noteFromMidiPitch(21), 'a0' , 'Note for Midi pitch 21 is a0')
```

Comment:
Again, I don't think an exhaustive test is needed here. Also test with and without `returnFlattened` set.